tdmpc2/common/scale.py

```diff
@@ -18,7 +18,7 @@ def load_state_dict(self, state_dict):
 
 	@property
 	def value(self):
-		return self._value.cpu().item()
+		return self._value
```

Comment:
avoid `cpu()` and `item()` which will not work with cudagraphs
`item()` causes a cuda synchronization, which is bad, and I think it also causes a graph break.